src/etc/htmldocck.py

```diff
@@ -440,6 +440,7 @@ def check_snapshot(snapshot_name, actual_tree, normalize_to_text):
 
         if bless:
             with open(snapshot_path, 'w') as snapshot_file:
+                actual_str = actual_str.replace(channel, "{{channel}}")
```

Comment:
What about only replacing `"https://doc.rust-lang.org/" + channel"`? I think that should drastically reduce the incidence of accidental substitution. (Unless there are other mentions of the channel that I'm not remembering, which is very possible!)

Reply:
`channel` is already a full URL (despite its name). E.g. it might be equal to  `"https://doc.rust-lang.org/nightly"` (and not just `"nightly"` defying intentions).